### PR TITLE
Readme updated

### LINKED ISSUE #2: Add basic how-to docu

### PR DESCRIPTION
**Pull request by derwasp**
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
-# Albelli.Extensions.Configuration.AmazonEC2ParameterStore
\ No newline at end of file
+# Albelli.Extensions.Configuration.AmazonEC2ParameterStore
```
Comment (A-KL) [inline README.md]: This line appears to be really huge eventually. I'd suggest to cut it out just to `AmazonEC2ParameterStore`

File: README.md [hunk 2]
```diff
@@ -1 +1,84 @@
...
+
+So
+`Configuration.GetSection("stage:api-lambda:s3")` and `Configuration.GetValue("stage:api-lambda:s3:bucketName")` should work as expected. 
+
+If you like to use `Configuration.GetSection(..).Bind()` or `Configuration.Bind()` - parameter names should not contain any extra characters, like `.-_`.
+
+Example usage:
+
+```
+
```
Comment (A-KL) [inline README.md]: An empty line. Do we need it?

File: README.md (hunk 1 above)
Comment (derwasp) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1 +1,84 @@
...
+        }
+
+        public IConfigurationRoot Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services
+                .AddSingleton<IMyAppSettings>(
+                    context => {
+                        var settings = new MyAppSettings();
```
Comment (A-KL) [inline README.md]: I'd also add as a part of this example `ConfigurationSection.Get<T>` extension. To show that the configuration code can be less ugly 😉 
```
        public static T Get<T>(this IConfigurationSection section)
            where T : class, new()
        {
            var settings = new T();

            section.Bind(settings);

            return settings;
        }

File: README.md [hunk 4]
```diff
@@ -59,12 +76,7 @@ namespace Albelli.AwesomeApi
         {
             services
                 .AddSingleton<IMyAppSettings>(
-                    context => {
-                        var settings = new MyAppSettings();
-                        var section = this.Configuration.GetSection($"{this.env}:{this.functionName}:settings");
-                        section.Bind(settings);
-                        return settings;
-                        })
+                    context => this.Configuration.GetSection($"{this.env}:{this.functionName}:settings").Bind<IMyAppSettings>())
```
Comment (A-KL) [inline README.md]: Suppose to be `.Get<IMyAppSettings>()` now 😉

Comment (A-KL) [reply README.md]: Was also thinking about having a variable above: `var section = this.Configuration.GetSection($"{this.env}:{this.functionName}:settings")` so the DI code will look simpler. But dosen't really matter.

**Pull request merged**